_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+The thing worse than searching while offline, or losing your connection to the Internet when doing it, is that NewPipe didn't let you type your search term.
+Being rudely interrupted by a network error is a thing of the past. Type all you want, and accept defeat after the fact. Both of these changes brought to you by [@Stypox](https://github.com/Stypox). Thanks!
 
-In order to save some of that valuable mobile data, you were already able to turn off video thumbnails and profile pictures for comments. Now even the black placeholders which replaced them are gone and comments can comfortably take up the space left behind. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
+To save valuable mobile data, turning off video thumbnails and profile pictures for comments was already possible. Now even the black placeholders which replaced them are gone, and comments can comfortably take up the space left over. Thanks, [@4D17Y4](https://github.com/4D17Y4)!
 
-[@TobiGr](https://github.com/TobiGr) brought two more improvements into this release: the setting to remember pop-up size and position has a less redundant description, and when you switch to another app, the main player now minimizes to background audio by default to allow for a more seamless transition. Swoosh.
+[@TobiGr](https://github.com/TobiGr) brought two more improvements: The setting to remember pop-up size and position has a less redundant description, and when switching to another app, the main player now minimizes to play audio in the background by default. Seamless transition. Swoosh.
 
-It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
+It is the current year, 2020, in higher resolution. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded last time, and will auto-select it when you hit "Download" on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
 
 Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
```

Comment:
```suggestion
Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for confusion.
```